src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +61,28 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// function to generate application details content
+const dynamicRowGeneration = (rowElementMaxCount, appDetailLabels, loading) => {
+  const labelArray = appDetailLabels.map(label => {
+    // only generate elements that are populated
+    if (!isEmpty(label[0])) {
+      // check to see if processing is required for output (booleans)
+      const spanValue = (typeof label[0] == 'boolean') ? label[0].toString() : label[0];
```

Comment:
JS has a very odd notion of equality. There are two operators, `==` and `===`. The latter (`===`) is what you would expect equality checks would do in other programming languages. The former(`==`) will try to run an equality check even if the two items are of different types, which leads to all sorts of weird results. 

In short, use `===`